warplib/ecrecover.cairo

```diff
@@ -20,12 +23,29 @@ func ecrecover_eth{range_check_ptr, bitwise_ptr : BitwiseBuiltin*, keccak_ptr :
     let (v_is_27) = warp_eq(v, 27)
     let (v_is_28) = warp_eq(v, 28)
 
+    # Expect one of them to be 1 (true)
     if v_is_27 + v_is_28 == 0:
```

Comment:
Should this be changed to `!= 1` for readability? Makes no difference though.

Reply:
In the context on of the early return on the next line, I think checking for 0 is more readable but I see your point as well. I think with the comment you added either of them are fine.